src/main/java/io/github/thebusybiscuit/mobcapturer/listeners/PelletListener.java

```diff
@@ -67,15 +68,24 @@ && canCapture(player, entity)
      */
     @ParametersAreNonnullByDefault
     protected boolean canCapture(Player p, LivingEntity entity) {
-        if (Slimefun.getProtectionManager().hasPermission(p, entity.getLocation(), Interaction.ATTACK_ENTITY)) {
-            if (MobCapturer.getRegistry().getConfig().getBoolean("options.capture-named-mobs")) {
-                return true;
-            } else {
-                return entity.getCustomName() == null;
-            }
-        } else {
+
+        if (!Slimefun.getProtectionManager().hasPermission(p, entity.getLocation(), Interaction.ATTACK_ENTITY)) {
+            //no permission check
+            return false;
+        }
+
+        List<String> ignoredMobNames = MobCapturer.getRegistry().getConfig().getStringList("options.ignored-mobs");
+        if (ignoredMobNames.contains(entity.getCustomName())) {
```

Comment:
what about color, upper/lower case

Reply:
This check is not included, it's just checking if the list contains the mob. I don't see a need to go that specific in detail.